admin/dw-options.php

```diff
@@ -58,6 +58,22 @@ function dw_render_options_page() {
 
 			<h2 class="title">Standaard waarden voor nieuwe cursussen</h2>
 			<table class="form-table">
+				<tr>
+					<th scope="row">
+						<label class="description" for="dw_settings[ccvCode]">
+							<?php _e('Training code'); ?>
```

Comment:
Why will `Training code` be translated, but the description not?

Reply:
Yes